.travis.yml

```diff
@@ -5,12 +5,14 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27,stack-scrapy-1.0,stack-scrapy-1.1,stack-scrapy-1.2,stack-scrapy-1.3,stack-scrapy-1.4,stack-scrapy-1.5
-    - python: 3.5
-      env: TOXENV=py35,stack-scrapy-1.1-py3,stack-scrapy-1.2-py3
-    - python: 3.6
-      env: TOXENV=py36,stack-scrapy-1.3-py3,stack-scrapy-1.4-py3,stack-scrapy-1.5-py3
+    - python: 3.5.2
+      env: TOXENV=py35,stack-scrapy-1.8-py3,stack-scrapy-2.0-py3,stack-scrapy-2.1-py3,stack-scrapy-2.2-py3,stack-scrapy-2.3-py3
+    - python: 3.6.1
+      env: TOXENV=py36,stack-scrapy-2.2-py3,stack-scrapy-2.3-py3,stack-scrapy-2.4-py3
+    - python: 3.7
+      env: TOXENV=py37,stack-scrapy-2.2-py3,stack-scrapy-2.3-py3,stack-scrapy-2.4-py3
     - python: 3.8
-      env: TOXENV=docs
+      env: TOXENV=py38,docs,stack-scrapy-2.2-py3,stack-scrapy-2.3-py3,stack-scrapy-2.4-py3
```

Comment:
I think we need a different PR for this test matrix updates.

Reply:
https://github.com/scrapy-plugins/scrapy-crawlera/pull/90